pyfr/solvers/aceuler/kernels/rsolvers/rsolve_trans.mako

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+<%include file='pyfr.backends.base.makocommon.transform'/>
+
+<%pyfr:macro name='rsolve' params='ul, ur, n, nf'>
+    fpdtype_t utl[${nvars}], utr[${nvars}], ntf[${nvars}];
+
+    utl[0] = ul[0]; utr[0] = ur[0];
+    ${pyfr.expand('transform_to','n', 'ul', 'utl', '1')};
```

Comment:
Space after ,

Is it possible to pass '1' as 1?

Reply:
I don't know if you remember but we tried this, but there were some issues with the pyfr was passing the stuff through mako. This seemed like the most straight forward fix as I really don't anticipate anyone really needing that functionality.

Reply:
Ah, yes, I recall this being a pain.